hermes-api/src/main/java/pl/allegro/tech/hermes/api/OfflineRetransmissionRequest.java

```diff
@@ -37,25 +41,25 @@ public OfflineRetransmissionRequest(
             @JsonProperty("endTimestamp") String endTimestamp) {
         this.sourceTopic = sourceTopic;
         this.targetTopic = targetTopic;
-        initializeTimestamps(startTimestamp, endTimestamp);
+        this.startTimestamp = initializeTimestamps(startTimestamp);
+        this.endTimestamp = initializeTimestamps(endTimestamp);
     }
 
-    private void initializeTimestamps(String startTimestamp, String endTimestamp) {
-        if (startTimestamp == null || endTimestamp == null) {
-            this.startTimestamp = null;
-            this.endTimestamp = null;
-            return;
+    private Instant initializeTimestamps(String timestamp) {
```

Comment:
nit: can be renamed to `initializeTimestamp` since there is only one function parameter